protocols/imap/src/main/java/org/apache/james/imap/processor/StoreProcessor.java

```diff
@@ -120,9 +120,6 @@ protected void processRequest(StoreRequest request, ImapSession session, Respond
                 if (messageSet != null) {
 
                     if (unchangedSince != -1) {
-                        // Ok we have a CONDSTORE option so use the CONDSTORE_COMMAND
-                        imapCommand = CONDSTORE_COMMAND;
```

Comment:
it looks weird, why you need to remove that?

Reply:
it was only set, never read